packages/data-context/src/sources/migration/codegen.ts

```diff
@@ -261,13 +261,13 @@ export async function hasSpecFile (projectRoot: string, folder: string, glob: st
 }
 
 export async function tryGetDefaultLegacyPluginsFile (projectRoot: string) {
-  const files = await globby('cypress/plugins/index.*', { cwd: projectRoot })
+  const files = await globby('cypress/plugins/index.*', { cwd: projectRoot, ignore: ['cypress/plugins/index.d.*'] })
```

Comment:
Changes look fine. Just curious though, why are we globbing this? We know where the the legacy configuration file is located and could pull this value from there and/or fallback to the previously file path.

Reply:
We know where it's located by default, but the default extension could be either `index.js`, `index.ts`, or `index.coffee`. So globbing it makes it easy to see if any of those are there.